code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -670,6 +670,12 @@
 	icon_state = "cyborg_upgrade3"
 	new_module = /obj/item/robot_module/security
 
+/obj/item/borg/upgrade/transform/enhancedengineering
+	name = "borg module picker (Engineering+)"
+	desc = "Allows you to give godlike powers to a engineering cyborg."
```

Comment:
```suggestion
	desc = "Allows you to give godlike powers to a engineering cyborg."
```
Inaccurate description.
Do not assume that people know what this module is. 
People should be able to understand what this upgrade is by reading the description instead of making the upgrade then being confused as to what it does when installed.

Reply:
reee alright